base/file.jl

```diff
@@ -441,3 +442,9 @@ function chmod(p::AbstractString, mode::Integer)
     uv_error("chmod",err)
     nothing
 end
+
+function chown(path::AbstractString, owner::Int, group::Int)
```

Comment:
Since `-1` leaves things unchanged I think it would make sense to switch `group` to be an optional parameter. Also owner and group should be `Integer`.
```julia
function chown(path::AbstractString, owner::Integer, group::Integer=-1)
    ...
end
```